addons/web_editor/static/lib/odoo-editor/src/utils/utils.js

```diff
@@ -963,7 +963,7 @@ export const formatSelection = (editor, formatName, {applyStyle, formatProps} =
                     tag.remove();
                     formatSpec.addStyle(getOrCreateSpan(selectedTextNode, inlineAncestors), formatProps);
                 }
-            } else {
+            } else if (!(formatName === 'fontSize' && formatProps.size === undefined)) {
```

Comment:
I think this is pretty hard to read. I think the following is more readable, what do you think ?
```js
formatName !== 'fontSize' || formatProps.size !== undefined
```

Also, could you please add some tests for this particular case ? One where there was already a style before (it should remove the existing style and there should be no empty span at the end) and one where there wasn't any style before (and therefore nothing should change).

Reply:
> ```js
> formatName !== 'fontSize' || formatProps.size !== undefined
> ```

Yes! this one is more readable. 👍